migration/node-operators.md

```diff
@@ -0,0 +1,93 @@
+# PoS Migration Guide for Node Operators
+
+Our migration guides are designed for users who want to actively participate in the transition process. This specific guide is for those who want to run the migration process without becoming validators (if you want to become one of the first validators from the start of the PoS chain, please refer to this [guide](migration-validators)). It is suitable for:
+
+- Node Operators
+- Exchanges
+- Regular Users
+
+This guide provides instructions to run the Activation Tool, the final step of the migration to the Nimiq PoS. There is no requirement to run the migration as a node operator. You can simply start your PoS client once the migration process ends and build your own node.
+
+It's important to note that, as a node operator, your role is primarily as an observer during the activation process. The migration will only proceed if at least 80% of the allocated stake is ready to migrate. Since you have not pre-registered as a validator, running the activation tool does not contribute to meeting the threshold requirement for the transition. Nonetheless, by running the migration tool, you will participate in the state migration and genesis block creation process.
```

Comment:
```suggestion
It's important to note that, as a node operator, your role is primarily as an observer during the activation process. The migration will only proceed if at least 80% of the allocated stake is ready to migrate. Since you have not pre-registered as a validator, running the activation tool does not contribute to meet the threshold requirement for the transition. Nonetheless, by running the migration tool, you will replicate the state migration and genesis block creation process.
```